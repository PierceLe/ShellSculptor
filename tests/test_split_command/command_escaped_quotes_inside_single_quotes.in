which 'I love \'INFO1112\''
