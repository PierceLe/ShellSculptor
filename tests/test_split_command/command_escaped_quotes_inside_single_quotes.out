>> which 'I love \'INFO1112\''
I love 'INFO1112' not found

